barretenberg/cpp/src/barretenberg/ultra_honk/protogalaxy.test.cpp

```diff
@@ -320,6 +320,7 @@ template <typename Flavor> class ProtoGalaxyTests : public testing::Test {
         auto second_accumulator = fold_and_verify(instances, composer, true);
         check_accumulator_target_sum_manual(second_accumulator, true);
 
+        // WORKTODO: looks like a bug/typo? should probably check second_accumulator
```

Comment:
yup, well spotted, technically it's fine cus `first_accumulator` is a valid accumulator but we need to check the second one